components_library/notes.md

```diff
@@ -0,0 +1,62 @@
+# Notes setting up RISC-V FS in components
```

Comment:
Lots of good stuff here, I'll add some of the more interesting ones as Jira issues this afternoon.